src/style.css

```diff
@@ -1 +1,21 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  flex: 0 0 auto;
+  height: 16px;
+  width: 16px;
+  background-image: url('images/star.svg');
+  background-size: contain;
```

Comment:
the background image should not stretch to the entire size of the element
check design